src/main/liquibase/liquibase-ciel-data-18x.xml

```diff
@@ -5,7 +5,7 @@
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```

Comment:
Please revert the changes to the data withing the file, i.e. this should still read `<comment>Preparing schema for adding mvp data</comment>`